components/needs_team/needs_team.jsx

```diff
@@ -80,6 +82,21 @@ export default class NeedsTeam extends React.Component {
 
         const team = this.updateCurrentTeam(this.props);
 
+        DesktopBridge.connect((event, data) => {
```

Comment:
Maybe destructure the `data` directly? `(event, {userIsActive})`?

Reply:
Redid the bridge ... this has changed. The new setup supports an arbitrary number of parameters.